### PR TITLE
Toward Arrow: kategory ank -> arrow ank

### PR DESCRIPTION
**Pull request by Guardiola31337**
- Migrates kategory `ank-core` module to arrow
- Adds `ank-gradle-plugin` module

👀 @raulraja

### CODE REVIEW

File: ank-core/src/main/kotlin/io/arrow/ank/main.kt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 @file:JvmName("main")
-package io.kategory.ank
+package io.arrow.ank
```
Comment (raulraja) [inline ank-core/src/main/kotlin/io/arrow/ank/main.kt]: `arrow` has no `io` package

File: ank-gradle-plugin/build.gradle [hunk 2]
```diff
@@ -0,0 +1,29 @@
+apply plugin: 'java-gradle-plugin'
+apply plugin: 'kotlin'
+
+//group = ext.ank_group
```
Comment (raulraja) [inline ank-gradle-plugin/build.gradle]: if thi sis no longer needed we can just :fire:

File: ank-gradle-plugin/gradle.properties [hunk 3]
```diff
@@ -0,0 +1,5 @@
+VERSION_NAME=0.1.7-SNAPSHOT
+POM_ARTIFACT_ID=ank-gradle-plugin
+POM_NAME=Ank Gradle Plugin
+POM_DESCRIPTION=Compile time docs verification for Kotlin (Gradle Plugin)
```
Comment (raulraja) [inline ank-gradle-plugin/gradle.properties]: verification & evaluation

**Comment by Guardiola31337** [comment]
@raulraja ready for a second round of 👀

**Pull request merged**